tests/integration/courses.test.js

```diff
@@ -186,67 +184,52 @@ describe('COURSES ROUTES - GET /courses/{_id}', () => {
     it('should get course', async () => {
       const response = await app.inject({
         method: 'GET',
-        url: `/courses/${courseId.authCompany.toHexString()}`,
+        url: `/courses/${courseIdFromAuthCompany.toHexString()}`,
         headers: { 'x-access-token': authToken },
       });
 
       expect(response.statusCode).toBe(200);
-      expect(response.result.data.course._id).toEqual(courseId.authCompany);
+      expect(response.result.data.course._id).toEqual(courseIdFromAuthCompany);
     });
   });
 
   it('should get course even if not authenticate', async () => {
     const response = await app.inject({
       method: 'GET',
-      url: `/courses/${courseId.authCompany.toHexString()}`,
+      url: `/courses/${courseIdFromAuthCompany.toHexString()}`,
     });
 
     expect(response.statusCode).toBe(200);
-    expect(response.result.data.course._id).toEqual(courseId.authCompany);
+    expect(response.result.data.course._id).toEqual(courseIdFromAuthCompany);
   });
 
-  describe('Other roles', () => {
-    it('should return 200 as user is course trainer', async () => {
-      authToken = await getTokenByCredentials(trainer.local);
+  const roles = [
+    { name: 'helper', expectedCode: 200 },
+    { name: 'auxiliary', expectedCode: 200 },
+    { name: 'auxiliary_without_company', expectedCode: 200 },
+    { name: 'coach', expectedCode: 200 },
+    { name: 'client_admin', expectedCode: 200 },
+    { name: 'training_organisation_manager', expectedCode: 200 },
+    { name: 'trainer', expectedCode: 200 },
+  ];
```

Comment:
Je suis desolee, je n'avais pas fait attention, mais vu que tout le monde peut recuperer les informations d'une formation (cf `it(should get course even if not authenticate)`) Est-ce que tester tous les roles est utile ?

Reply:
jsp :)

Reply:
bien vu chloé je ne pense pas ;)